internal/tracker/tracker_test.go

```diff
@@ -158,10 +167,107 @@ task-bundles:
 				_, err = f.WriteString(tt.input)
 				assert.NoError(t, err)
 			}
-			output, err := Track(context.TODO(), tt.urls, "pipeline-bundles", inputFile)
+			output, err := Track(context.TODO(), tt.urls, inputFile, func(ctx context.Context, ref image.ImageReference) ([]string, error) {
+				return []string{"pipeline"}, nil
+			})
 			assert.NoError(t, err)
 			assert.Equal(t, tt.output, string(output))
 		})
 	}
 
 }
+
+func TestTrackerAddKeepsOrder(t *testing.T) {
+	parameters := gopter.DefaultTestParameters()
+
+	// uncomment when test fails and set the seed to the printed value
```

Comment:
Is this leftover?

Reply:
When generative tests fail they print the random seed, with this random seed one can reproduce that exact case, this is code that does that, it needs to be uncommented and the seed value needs to be placed, I thought the comment explained that

Reply:
It makes sense now. Thanks!